package.json

```diff
@@ -52,10 +71,9 @@
     "webrtcsupport": "^2.2.0"
   },
   "dependencies": {
+    "@libp2p/logger": "^1.0.2",
     "abortable-iterator": "^3.0.0",
-    "class-is": "^1.1.0",
```

Comment:
I assumed that `class-is` is not required if the project is migrated to Typescript. Am I correct?